bootstrapvz/common/fs/loopbackvolume.py

```diff
@@ -15,7 +15,7 @@ def _before_create(self, e):
         log_check_call(['truncate', size_opt, self.image_path])
 
     def _before_attach(self, e):
-        [self.loop_device_path] = log_check_call(['losetup', '--show', '--find', self.image_path])
+        [self.loop_device_path] = log_check_call(['losetup', '--show', '--find', '--partscan', self.image_path])
```

Comment:
Since Util-linux 2.21, losetup is able to scan the partition table of the newly created loop device. Therefore it is able to unmap them as well. By adding a ```partscan```, losetup tries to map the partitions when the loop device is mounted, and will unmap them when it is detached.